### ISSUE TITLE
[d3d9] - DARK SOULS™ II (236430) - Grass in "Things Betwixt" does not render (most of the time).

### ISSUE THREAD

**Issue by ZereoX** [doitsujin/dxvk#2107]
### Description:
In **_DARK SOULS™ II_** (236430) grass in **Things Betwixt** (starting location) does not render, but some will ocasionally pop-in depending on distance and camera angle.

- Issue is also present on WineD3D
- Issue occurs on both Nvidia and AMD GPUs.
- Issue occurs on both Proton 6.3-4 and Windows on all major DXVK versions since 1.5, including master (6b8ab4fd385cc3cdeb272e4b4a6b69a1d4100d59). 

| Windows - DXVK 1.9 | Proton - WineD3D | Windows - Native
| ---- | ---- | ---- |
|![Windows - DXVK](https://user-images.githubusercontent.com/761853/122217477-738f4300-ce7b-11eb-94f4-924828769262.png)|![Proton - WineD3D](https://user-images.githubusercontent.com/761853/122217527-80139b80-ce7b-11eb-8af1-2ee9f4e53a69.png)|![Windows - Native](https://user-images.githubusercontent.com/761853/122217554-83a72280-ce7b-11eb-8260-dd40a7e45b1f.png)|

Please let me know if any additional information is required or if additional testing is needed.

#### Notes
- Issue occurs at all graphical preset.
- Issue is not present in DARK SOULS™ II: Scholar of the First Sin (uses DirectX 11 instead of 9).

### Software information
[DARK SOULS™ II](https://store.steampowered.com/app/236430/DARK_SOULS_II/) (236430)

### System information
OS: Ubuntu 20.04 / Windows 10 Pro 20H2
GPU: Radeon RX 5700 XT / Nvidia GeForce GTX 1060
Driver: 21.2\~git2106150600.639579\~oibaf\~f / 465.27-0ubuntu0.20.04.2
Wine version: Proton 6.3-4
DXVK version: 1.5.5, 1.6.1, 1.7.3, 1.8.1, 1.9

### Apitrace file
- Link: https://drive.google.com/file/d/13j_-lX7BoJ0AHcGm_vbr6LNvwwZgtcUC/view?usp=sharing

### Log files
d3d9.log: [DarkSoulsII_d3d9.log](https://github.com/doitsujin/dxvk/files/6662914/DarkSoulsII_d3d9.log)

```
warn:  ConvertFormat: Unknown format encountered: D3D9Format::AL16
warn:  ConvertFormat: Unknown format encountered: D3D9Format::R16
warn:  D3D9: VK_FORMAT_D24_UNORM_S8_UINT -> VK_FORMAT_D32_SFLOAT_S8_UINT
warn:  D3D9DeviceEx::SetRenderState: Unhandled render state D3DRS_MULTISAMPLEANTIALIAS
```

**Comment by doitsujin** [comment]
Can you try Windows + DXVK?

**Comment by K0bin** [comment]
Should be fixed by c13395db970d16a5631a7c0926e2c4529d2710c3
